src/Symfony/Bridge/Twig/Command/DebugCommand.php

```diff
@@ -12,11 +12,13 @@
 namespace Symfony\Bridge\Twig\Command;
 
 use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidArgumentException;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\Console\Style\SymfonyStyle;
+use Symfony\Component\Finder\Finder;
```

Comment:
Finder is currently not a dep of Twig bridge.

Reply:
All dependencies, including `symfony/finder` are in `require-dev` section:
https://github.com/symfony/symfony/blob/b67c26c2efe4673553d6379367ebecc2eac7dca5/src/Symfony/Bridge/Twig/composer.json#L23-L30

Note that the `Finder` component was already used in other places like `TwigExtractor` and `LintCommand`, do we need to do something else?